disttask/importinto/dispatcher.go

```diff
@@ -220,6 +221,10 @@ func (dsp *importDispatcherExt) OnNextStage(ctx context.Context, handle dispatch
 	var nextStep int64
 	switch gTask.Step {
 	case proto.StepInit:
+		if metrics, ok := metric.GetCommonMetric(ctx); ok {
```

Comment:
Why not add this counter when generating the taskMeta.Plan?

Reply:
you mean on creating task? we only want this metric after the task start running